Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -291,18 +291,24 @@ def condition_(self, fragment, stepList, key, hasHarvest):
 
 class UpgradeWorkflow_trackingMkFit(UpgradeWorkflowTracking):
     def setup_(self, step, stepName, stepDict, k, properties):
+        if 'Digi' in step: stepDict[stepName][k] = merge([self.step2, stepDict[step][k]])
```

Comment:
not sure I understand this change :(

Reply:
It's copied from the similar line below for Reco. IIUC something like that is needed to add the `--customise` command to the cmsDriver line.